staging/src/k8s.io/legacy-cloud-providers/openstack/MAINTAINERS.md

```diff
@@ -3,4 +3,4 @@
 * [Angus Lees](https://github.com/anguslees)
 
 
-[![Analytics](https://kubernetes-site.appspot.com/UA-36037335-10/GitHub/pkg/cloudprovider/providers/openstack/MAINTAINERS.md?pixel)]()
```

Comment:
I don't think this is intentional?

Reply:
I have no idea why there's a tracking pixel back to google cloud in there.